### PR TITLE
Use w3c/charter-draft's template

### PR DESCRIPTION
**Pull request by csarven**
Resolves https://github.com/solid/solid-wg-charter/issues/2

---

I've applied the template from charter-drafts as best as possible without touching the core content.

I believe the simplest way to diff the content changes (for errors) is to visually compare

http://htmlpreview.github.io/?https://github.com/solid/solid-wg-charter/blob/1edb984622800bdcc0c49909cd239596374e932b/charter/index.html (this PR)

with:

https://solid.github.io/solid-wg-charter/charter/ (main branch on gh-pages)

(or perhaps copy text to clipboard from what's currently rendered and search for that text in this PR's rendered version)

See also W3C's HTML diff:

https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fsolid.github.io%2Fsolid-wg-charter%2Fcharter%2F&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fsolid-wg-charter%2F1edb984622800bdcc0c49909cd239596374e932b%2Fcharter%2Findex.html

May also want to diff with https://w3c.github.io/charter-drafts/charter-template.html to ensure that the template is accurate:

https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2Fcharter-template.html&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fsolid-wg-charter%2F1edb984622800bdcc0c49909cd239596374e932b%2Fcharter%2Findex.html

Any changes to charter-template.html at w3c/charter-drafts should be easy to integrate if needed.

---

Some parts of the charter can use a review based on this template. For example the deliverables section with the state of a specification that's being taken on. I left the existing content as is, figured that we can update whatever needs to change in separate PR or possible proposed changed here, e.g., the charter names "Solid Protocol v1.0" but I believe what's intended in the template is the current name/state of the specification, and not the target. Hence, I suggest changing something like that to "Solid Protocol" or "Solid Protocol v0.10.0".

The timeline format can also be revised. It is a list as opposed to a table. I didn't touch that either.

There may be other minor things but I think this is pretty safe. Please have a look.

### CODE REVIEW

File: charter/index.html [hunk 1]
```diff
@@ -1,303 +1,245 @@
...
+          <h3>
+            Normative Specifications
+          </h3>
+          <p>
+            The Working Group will deliver the following W3C normative specifications:
+          </p>
                 <dl>
-                    <dt id="protocol" class="spec">Solid Protocol v1.0</dt>
+                    <dt id="solid-protocol" class="spec"><a href="https://solidproject.org/TR/protocol">Solid Protocol v1.0</a></dt>
                     <dd>
```
Comment (pchampin) [inline charter/index.html:243]: The Solid TR document is not the deliverable of the WG. This link is misleding. I replaced it with an explicit reference, for the deliverable, to the current specification (but seen as a separate document).
```suggestion
                    <dt id="solid-protocol" class="spec">Solid Protocol v1.0</dt>
                    <dd>
                    <p><strong>Reference:</strong> <a href="https://solidproject.org/TR/protocol">Solid Protocol v.0.10.0</a>, eds. Sarven Capadisli, Tim Berners-Lee, Ruben Verborgh, Kjetil Kjernsmo. Solid Technical Report 2023.</p>
```

**Comment by elf-pavlik** [comment]
It looks like the content of some sections has also changed. For example:

* Participation
* Communication
* Decision Policy

I think it would be much easier to review if this PR only had structural changes and any changes to the content are submitted as separate PRs.

**Review by pchampin — CHANGES_REQUESTED** [review]
Good to go except for the link on 'Solid Protocol' in the Deliverable section, which IMO conflates the CG document and the future WG document.

**Comment by pchampin** [comment]
> It looks like the content of some sections has also changed. For example:
> 
>     * Participation
> 
>     * Communication
> 
>     * Decision Policy
> 
> 
> I think it would be much easier to review if this PR only had structural changes and any changes to the content are submitted as separate PRs.

The parts that are modified are in fact boilerplate text that was kept unchanged from the old template, and that @csarven replaced with the boilerplate from the new template.

**Review by elf-pavlik — APPROVED** [review]

**Review by pchampin — APPROVED** [review]

**Pull request merged**